mysite/missions/svn/tests.py

```diff
@@ -33,18 +34,16 @@
     TestCase, subproc_check_output, make_testdata_filename)
 from mysite.missions.svn import views
 from mysite.missions.svn import view_helpers
-from django.utils.unittest import skipIf
 import mysite.base.depends
 from mysite.missions.svn.forms import DiffForm
 
 
-@skipIf(not mysite.base.depends.svnadmin_available(), (
-        "Skipping tests for Subversion training mission for now. To run "
-        "these tests, install the 'subversion' package in your package "
-        "manager."))
+@skipIf(not mysite.base.depends.svnadmin_available(),
+    ("Skipping tests of Subversion backend. Install the 'subversion' to run."))
```

Comment:
For some reason I am chuckling at INSTALL THE 'SUBVERSION'. I think this would read better with "package"?